src/components/pokemon/Pokemon.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+import './Pokemon.css';
+
+function Pokemon({name}) {
+    const [pokemonData, setPokemonData] = useState({});
+
+    useEffect(() => {
+        async function fetchPokemon() {
+            try {
+                const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`);
+                setPokemonData(response.data);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchPokemon();
+
+    }, []);
+
+
+    return (
+
+        <section className="poke-card">
+            {pokemonData &&
+                <>
+                    {Object.keys(pokemonData).length > 0 &&
```

Comment:
Deze check kan dan iets simpeler
```suggestion
                    {pokemonData &&
```